ansible/validate.yml

```diff
@@ -27,4 +27,26 @@
     - import_role:
         name: opendistro
         tasks_from: validate.yml
-      tags: validate
\ No newline at end of file
+      tags: validate
+
+- name: Validate rebuild configuration
+  hosts: openhpc
```

Comment:
This looks a bit openstack specific. Is there a way we could only run this only if we are using openstack?

Reply:
Yeah or possibly just a variable that sets the cloud type? E.g
````
appliance_cloud_type: openstack
```

Reply:
Have gone with a group as its consistent with all other features and there could be reasons why you only want to enable rebuild on a subset of nodes I guess.